tests/SentryMiddlewareTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Sentry\Tests;
+
+use HttpSoft\Message\Response;
+use HttpSoft\Message\ServerRequest;
+use PHPUnit\Framework\Error\Error;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use RuntimeException;
+use Yiisoft\Yii\Sentry\SentryMiddleware;
+use Yiisoft\Yii\Sentry\Tests\Stub\Transport;
+
+final class SentryMiddlewareTest extends TestCase
+{
+    public function testProcessWithException(): void
+    {
+        $methodName = debug_backtrace()[0]['function'];
+        $eventKey = self::class . "::$methodName()";
+
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage('Exception test.');
+
+        try {
+            $middleware = new SentryMiddleware($this->createSentryHub($eventKey));
+            $middleware->process(
+                new ServerRequest(method: 'GET', uri: '/'),
+                $this->createRequestHandlerWithException(),
+            );
+        } catch (RuntimeException $e) {
```

Comment:
```suggestion
        } catch (Throwable $e) {
```
Catch all exception types to make sure you won't skip anything

Reply:
This is not the case, please see https://github.com/yiisoft/yii-sentry/pull/15#discussion_r925547207,